f5_sphinx_theme/head.html

```diff
@@ -17,13 +17,15 @@
   <!-- Twitter Bootstrap viewport setting -->
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <!-- F5 metadata -->
-  <meta name="kb_doc_type" content="Manual"/>
-  <meta name="kb_doc_type" content="User Guide"/>
   <meta name="title" content="{{ title|striptags|e }}"/>
-  <meta name="product" content="{{ project|striptags|e }}"/>
+  <meta name="product" content="{{ project|striptags|e }}"/> -
   <meta name="version" content="{{ release|striptags|e }}"/>
-  <meta name="updated_date"  content="{{ last_updated|e }}"/>
-  <meta name="archived" content="Archived documents excluded"/>
+  <meta name="updated_date"  content="{{ last_updated|e }}"/> -
+  <meta name="archived" content="Archived documents excluded"/> -
+  <meta name="doc_type" content="Manual"/>
+  <meta name="lifecycle" content="release"/>
+  <meta name="F5 Cloud Services" content="1.0"/>
+
 {% if (theme_version_selector != 'False') %}
```

Comment:
Need to add this one: 
    <meta name="{{ project|striptags|e }}" content="{{ release|striptags|e }}"/>
Which will result in something like:
   <meta name="F5 Cloud Services" content="1.0"/>

They require one tag that is a combo of the product and version.

Reply:
Looks like you have to click Edit to see what I pasted--